### ISSUE TITLE
Add USWDS skipnav above gov website banner

### ISSUE THREAD

**Issue by drew-usds** [CDCgov/prime-reportstream#2772]
** Research and implement if low effort, if high effort discuss with team **

**Describe the bug**
As a user with a screen reader, I should be able to skip over the "This is an official gov website ..." banner and accordion with an anchor link and get right to the content.

Currently exists on 11ty site under `_includes/banner.liquid`. I would have just *added it* instead of creating this bug ticket, but I'm not totally sure how the gov banner component is being pulled in now.

**Impact**
Is a USWDS recommended accessibility feature.

**To Reproduce**

**Expected behavior**
Banner markup should look like:
```
<a class="usa-skipnav" href="#main-content">Skip to main content</a>
<section class="usa-banner" aria-label="Official government website"> ... </section
```

**Screenshots**

**Logs**
If applicable, please attach logs to help describe your problem.

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by HeatherRienks** [comment]
Per 12/126 Eng ticket mt, we will take this on

**Comment by kevinhaube** [comment]
To prevent myself from having to go back and change this to use `USLink`, I'm going to wait until #7796 is merged

**Comment by kevinhaube** [comment]
@chris-kuryak @stephenkao 
Adding the `release_notes` label: once merged and deployed to `production`, the website will have a "skip navigation" link that takes users with screen readers to the main page content instead of tabbing through each nav item. Accessibility win!

**Comment by kevinhaube** [comment]
Set to deploy Thursday, Jan 6!
@chris-kuryak Will be good for January 2023 Release Notes. See comment above for details.

**Comment by chris-kuryak** [comment]
@kevinhaube Thanks! added as a comment to the Jan 2023 release notes ticket as a reminder.
